Quran/AyahInfoPersistenceStorage.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  AyahInfoPersistenceStorage.swift
+//  Quran
+//
+//  Created by Ahmed El-Helw on 5/12/16.
+//  Copyright © 2016 Quran.com. All rights reserved.
+//
+
+import Foundation
+import SQLite
+
+struct AyahInfoPersistenceStorage {
```

Comment:
Can you create a protocol for `AyahInfoPersistenceStorage` then making that struct conforms to it with a name like `SQLiteAyahInfoPersistenceStorage`? Then we can dependency inject it to the AyahRetriever instea of depending on implementation.